tests/AngleRotaionTest.cpp

```diff
@@ -0,0 +1,231 @@
+/**
+ * @file   AngleRotationTest.cpp
+ * @brief  AngleRotationクラスのテスト
+ * @author Hara1274
+ */
+
+#include "AngleRotation.h"
+#include <gtest/gtest.h>
+#include <gtest/internal/gtest-port.h>
+#include "Robot.h"
+#include "SystemInfo.h"
+
+namespace etrobocon2024_test {
+
+  // @see https://github.com/KatLab-MiyazakiUniv/etrobocon2022/blob/main/docs/odometry.md
+  constexpr double TRANSFORM = 2.0 * WHEEL_RADIUS / TREAD;  // 回頭角度を求める式の係数
+
+  // 右回頭のテスト
+  TEST(AngleRotationTest, runRight)
```

Comment:
テスト名アッパーキャメルでお願いします🙇以降も同様
```suggestion
  TEST(AngleRotationTest, RunRight)
```